example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,45 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(collection, ['createdAt', 'updatedAt']));
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      const originalCollection = collection;
+      const updatedCollection = {
+        ...originalCollection,
+        reportToEms: false,
+      };
+      await updateCollection({
+        prefix: config.stackName,
+        collection: updatedCollection,
+      });
+
+      await expectAsync(waitForObjectToExist({
+        bucket: config.bucket,
+        key: recordUpdatedKey,
+      })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordUpdatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Update');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(updatedCollection, ['createdAt', 'updatedAt']));
+      expect(originalCollection.reportToEms).toEqual(true);
```

Comment:
We should move this up to be a comparison against the record retrieved from the API (see comment above), rather than against the fixture we sent to the API